### DISCUSSION TITLE
what is the best way to run multiple test files?

### DISCUSSION THREAD

**Discussion by omariosouto** [babashka/nbb#226] [Q&A] [answered]
I'm creating a small/medium size project and this question came to my mind, there's a way to autoload everything like I'm doing here but better?

https://github.com/devsoutinho/labs/pull/9/files#diff-a9a4ca6e49afa91ffc74b1e1d1fd973db068da4f17a20b016420a18582503ecb

**Comment by borkdude** [comment, marked as answer]
That looks about right, although I have doubts about:

```
(mapv t/run-tests test-files)
```

I think this should read as:

```
(apply t/run-tests test-files)
```

There is currently no built-in test runner which auto-discovers files.

**Reply by omariosouto** [reply]
Okay! I will do a "temporary" solution by trying to discover the files by myself and put them here, thanks by the answer \o

I know that in Clojure community is more common have a separated folder for the tests but the approach that I'm following here could be a great option also considering that a company could have multiple nbb scripts and run all of them separated in a CI pipeline
